liquibase-core/src/main/java/liquibase/change/core/AddAutoIncrementChange.java

```diff
@@ -186,11 +195,13 @@ public ChangeStatus checkStatus(Database database) {
             }
 
             if (getGenerationType() != null && column.getAutoIncrementInformation().getGenerationType() != null) {
-                result.assertCorrect(getGenerationType().equals(column.getAutoIncrementInformation().getGenerationType()), "Generation type is incorrect");
+                result.assertCorrect(getGenerationType().equals(column.getAutoIncrementInformation().getGenerationType()),
+                     "Generation type is incorrect");
             }
 
             if (getDefaultOnNull() != null && column.getAutoIncrementInformation().getDefaultOnNull() != null) {
-                result.assertCorrect(getDefaultOnNull().equals(column.getAutoIncrementInformation().getDefaultOnNull()), "Default on null is incorrect");
+                result.assertCorrect(getDefaultOnNull().equals(column.getAutoIncrementInformation().getDefaultOnNull()),
+                     "Default on null is incorrect");
```

Comment:
@adrian-velonis1, hi! Is the extra spacing here required for autogenerating the documentation? I'm curious because it is an unusual way to see the code formatted.

Reply:
@XDelphiGrl The indenation isn't technically required, I just wanted to avoid having to do a lot of left/right scrolling to edit the change type descriptions. Normally I would put `description = "blah"` on a new line only if it was getting close to 120ish characters.

This particular one for `getDefaultOnNull()` does look a little strange and was possibly an accident. At least I don't think this particular string would be something we'd grab for the docs.